lib/service/kubernetes.go

```diff
@@ -236,6 +237,7 @@ func (process *TeleportProcess) initKubernetesService(log *logrus.Entry, conn *C
 		DynamicLabels:        dynLabels,
 		CloudLabels:          process.cloudLabels,
 		Log:                  log,
+		PROXYProtocolMode:    multiplexer.PROXYProtocolOff,
```

Comment:
Can you add a comment here explaining why this is always "off" and doesn't take value from config.